api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -553,7 +563,20 @@ private void createConfigFile(org.pmiops.workbench.firecloud.model.Workspace fcW
     fireCloudService.cloneWorkspace(workspaceNamespace, workspaceId,
         fcWorkspaceId.getWorkspaceNamespace(), fcWorkspaceId.getWorkspaceName());
 
-    // TODO(calbach): Determine whether we need to copy GCS notebooks here.
+    String toBucket = null;
+    try {
+      toBucket = fireCloudService.getWorkspace(
+          fcWorkspaceId.getWorkspaceNamespace(), fcWorkspaceId.getWorkspaceName())
+          .getWorkspace()
+          .getBucketName();
+    } catch (ApiException e) {
+      log.log(Level.SEVERE, "Firecloud error retrieving newly cloned workspace", e);
+      throw new ServerErrorException();
+    }
+
+    for (String name : cloudStorageService.getBucketFileList(fromBucket, "notebooks")) {
```

Comment:
We should either make "notebooks" a constant or a config setting.

I'm wondering if we should have a filter on file extension here? Or max size? (We don't want to copy over ginormous files probably?)

One thing we'll also need to think about is whether outputs embedded in notebooks should be cloned along with the notebook code... along with how syncing files to GCS works.

Reply:
Note that this setting ("notebooks") should be the same one that Neha uses to localize notebooks from GCS to the cluster.